Library/Formula/symlinks.rb

```diff
@@ -0,0 +1,23 @@
+class Symlinks < Formula
+  desc "scan/change symbolic links"
+  homepage "http://www.ibiblio.org/pub/Linux/utils/file/symlinks.lsm"
```

Comment:
> Well, seems like I have been both confident and wrong at the same time.

I can't complain; I'm guilty of this all the time. Thanks for double checking! :heart: